editor/import/editor_import_plugin.cpp

```diff
@@ -112,6 +112,14 @@ int EditorImportPlugin::get_import_order() const {
 	ERR_FAIL_V_MSG(-1, "Unimplemented _get_import_order in add-on.");
 }
 
+int EditorImportPlugin::get_format_version() const {
+	int ret;
+	if (GDVIRTUAL_CALL(_get_format_version, ret)) {
+		return ret;
+	}
+	ERR_FAIL_V_MSG(-1, "Unimplemented _get_format_version in add-on.");
```

Comment:
This will cause all existing implementations of `EditorImportPlugin` to produce an error message as well as a -1 return here, which will probably break some addons.

Currently, core/io/resource_importer.h has `virtual int get_format_version() const { return 0; }` which should apply as a default to user-made EditorImportPlugin's.

I would prefer to keep the current behavior of returning 0 when `_get_format_version` is not implemented / supported, and perhaps suppress the error message, or perhaps a `WARN_ONCE` so users have some notice that it's worth implementing this.

Reply:
Thanks!

That makes sense - in my latest push, I've updated it to just return `0` if the virtual method isn't implement. This is different than the other virtual methods (which all include an error like I previously had here), but for compatibility, I agree that no message is probably best.